src/nft/components/collection/MarketplaceSelect.tsx

```diff
@@ -21,7 +21,7 @@ const FilterItemWrapper = styled(Row)`
   cursor: pointer;
   border-radius: 12px;
   &:hover {
-    background: ${({ theme }) => theme.backgroundInteractive};
+    background: ${({ theme }) => theme.surface1};
```

Comment:
Don't know exactly what this is touching so I'm not sure

Reply:
definitely at least surface2, or 3.

heres 2:
<img width="373" alt="CleanShot 2023-08-17 at 10 55 20@2x" src="https://github.com/Uniswap/interface/assets/12100/41da8ef9-9e58-4fba-83d2-493721d35405">

heres 3:
<img width="363" alt="CleanShot 2023-08-17 at 10 55 49@2x" src="https://github.com/Uniswap/interface/assets/12100/0e1a1353-b8b2-4563-84ae-c960f32d08b9">

3 matches the hover of the parent group, but actually 2 doesn't look bad because its nested inside the parent so a bit less bold, but either seem fine

Reply:
let's do 2 please

Reply:
done